python/api/audit/urls.py

```diff
@@ -11,15 +11,10 @@
 # limitations under the License.
 
 
-from django.urls import path
+from api.audit.views import AuditLoginViewSet, AuditOperationViewSet
 from rest_framework.routers import SimpleRouter
 
-from .views import AuditOperationViewSet, AuditLoginViewSet, AuditRoot
-
 router = SimpleRouter()
-router.register(r'operation', AuditOperationViewSet, basename='audit-operations')
-router.register(r'login', AuditLoginViewSet, basename='audit-logins')
-urlpatterns = [
-    path(r'', AuditRoot.as_view(), name='root'),
-    *router.urls,
-]
```

Comment:
As I understand, without registering `AuditRoot`, it'll not be shown in viewer in API root, but operation/ and login/ will be available through direct access.
@a-alferov Do we want that?
It looks like it doesn't break ADCM Client thou

Reply:
Reverted